src/renderer/login/components/LoginFormLedger/LoginFormLedger.js

```diff
@@ -61,12 +60,9 @@ export default class LoginFormLedger extends React.PureComponent {
 
   renderActions = () => {
     const disabled = this.props.disabled || this.props.progress !== LOADED;
-    const onClick = disabled ? null : this.handleLogin;
```

Comment:
Whoops, looks like the disabled state of this wasn't consistent.  Nice cleanup here!